include/albatross/src/utils/async_utils.hpp

```diff
@@ -32,10 +32,10 @@ template <typename ValueType, typename ApplyFunction,
                                       ApplyFunction, ValueType>::value &&
                                       std::is_same<void, ApplyType>::value,
                                   int>::type = 0>
-void async_apply(const std::vector<ValueType> &xs, const ApplyFunction &f) {
+void async_apply(const std::vector<ValueType> &xs, const ApplyFunction &func) {
```

Comment:
"f" was in use during the loop over futures, giving two variables in scope the same name. While this worked and compiled, it gave compiler warning with verbosity.